src/stories/Components/Button/Button.mdx

```diff
@@ -56,9 +57,9 @@ Button can be used as a standalone component, however, it does have a few depend
 
 ### No Border
 
-`.btn--no-border` This is a modifier which can be used with all button classes except delete. It provides a button with no border, but the same sizing and adapts it's color to all the btn variations.
+`.btn--no-border` This is a modifier which can be used with all button classes except destructive. It provides a button with no border, but the same sizing and adapts it's color to all the btn variations.
 
-Note: `.btn-delete` and `.btn-warning` do not support the `.btn--no-border` modifier. Borderless warning or delete buttons is not considered a good pattern and can easily lead to confusion so it has been disabled.
+Note: `.btn-destructive` and `.btn-warning` do not support the `.btn--no-border` modifier. Borderless warning or destructive buttons is not considered a good pattern and can easily lead to confusion so it has been disabled.
```

Comment:
I wasn't aware this was a bad pattern. Is there some more documentation or info someone can point me towards that explains this? Or can one of y'all explain?

Reply:
I think the general idea was that we want to call attention to destructive actions to make it more obvious, but then again, not always calling attention to them might be beneficial.

I wonder if we should rethink this at some point (post v1, doesn't seem mission critical atm)

From what i can find, Material 3 doesn't specifically talk about destructive actions and some of the dialog examples of deleting show text only.
https://m3.material.io/components/dialogs/guidelines#4fe52c7a-64e8-4e80-bb6e-40f428f17b6c Shows a full button
https://m3.material.io/components/dialogs/guidelines#5c70da4d-3a2f-4271-b36c-16769a91a474 Shows normal text I'm assuming with a hover state.

HIG has a few things on buttons and destructive actions, but I could not find any guidance on text vs background
https://developer.apple.com/design/human-interface-guidelines/buttons#Role

https://uxmovement.com/buttons/how-to-design-destructive-actions-that-prevent-data-loss/
This paragraph specifically:

## Reserve Red Warning Signals for Confirmations
When the destructive action is on a confirmation screen, you want the user’s full attention. Conversely, when the destructive action is not on a confirmation screen, you don’t want to call attention to it. Doing so compels them to press it when they have no intention to commit an act of destruction."